.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: build
+
+on:
+  push:
+    branches:
+      - release/**
```

Comment:
Why not `master` too for "nightly" builds?

Reply:
Not worth the trouble right now, this repo gets hardly any traffic.